src/aspire/commands/preprocess.py

```diff
@@ -0,0 +1,76 @@
+import logging
+
+import click
+
+from aspire.estimation.noise import WhiteNoiseEstimator
+from aspire.source.relion import RelionSource
+
+logger = logging.getLogger(__name__)
+
+
+@click.command()
+@click.option('--data_folder', default=None,
+              help='Path to data folder')
+@click.option('--starfile_in', required=True,
+              help='Path to input starfile relative to data folder')
+@click.option('--starfile_out', required=True,
+              help='Path to output starfile relative to data folder')
+@click.option('--pixel_size', default=1, type=float,
+              help='Pixel size of images in starfile')
+@click.option('--max_rows', default=None, type=int,
+              help='Max. no. of image rows to read from starfile')
+@click.option('--flip_phase', default=True,
+              help='perform phase flip or not')
+@click.option('--max_resolution', default=16, type=int,
+              help='Resolution of downsampled images read from starfile')
+@click.option('--normalize_background', default=True,
+              help='Normalize images to background noise or not')
+@click.option('--whiten_noise', default=True,
+              help='whiten background noise')
+@click.option('--invert_contrast', default=True,
+              help='Invert the contrast of images so molecules are shown in white')
+@click.option('--batch_size', default=512,
+              help='batch size to perform pipeline so that not crash memory')
+@click.option('--save_mode', default='single',
+              help='Option to save mrc file, if not single, saved to multiply files in batch size')
+@click.option('--overwrite', default=False,
```

Comment:
Whether to overwrite mrc files if they already exist

Reply:
Done in commit 0b58b0d.